command/server/config/config.go

```diff
@@ -123,6 +124,7 @@ func DefaultConfig() *Config {
 		LogFilePath:              "",
 		JSONRPCBatchRequestLimit: DefaultJSONRPCBatchRequestLimit,
 		JSONRPCBlockRangeLimit:   DefaultJSONRPCBlockRangeLimit,
+		VerifyOutcomeApi:         "",
```

Comment:
See previous comment, I think it would be better to have this within the DataFeed config
d9ca26fd9910b047e41e29abbab91ef2df200f6595782f5e60de4652c78541caR116